dist/styles.css

```diff
@@ -0,0 +1,5 @@
+.moxy-next-link {
+    color: unset; /* stylelint-disable-line */
+    text-decoration: none;
```

Comment:
by default this makes more sense having text-decoration, mainly because link in body text can have some visual feedback like an underline.

https://codepen.io/nlfonseca/pen/GRpZKoG

Reply:
Agreed.

Having said that, the CSS will look like this:

```css
.moxy-next-link {
    color: inherit;
}
```

Do you guys think it still makes sense to include this CSS file? As this will almost definitely be styled by the developer, I think we should skip it.

Reply:
Agree!

Reply:
I agree with @nlfonseca , we shouldn't be removing the `text-decoration`. I also agree with @fsdiogo: imo this file is unnecessary.